src/dotnet/Users.Contracts/Backend/IUserProfilesBackend.cs

```diff
@@ -5,18 +5,11 @@ public interface IUserProfilesBackend
     [ComputeMethod(KeepAliveTime = 10)]
     Task<UserProfile?> Get(string id, CancellationToken cancellationToken);
     [ComputeMethod(KeepAliveTime = 10)]
-    Task<UserProfile?> GetByName(string name, CancellationToken cancellationToken);
```

Comment:
Let's also remove client defs

Reply:
Updated IUserProfilesBackendClientDef. But it seems it's not used for now.